cmd/protoc-gen-go-grpc/main.go

```diff
@@ -34,19 +34,17 @@ import (
 	"flag"
 
 	"google.golang.org/protobuf/compiler/protogen"
-	"google.golang.org/protobuf/types/pluginpb"
 )
 
-var genUnstableServerInterfaces *bool
+var requireUnimplemented *bool
 
 func main() {
 	var flags flag.FlagSet
-	genUnstableServerInterfaces = flags.Bool("gen_unstable_server_interfaces", false, `set to generate legacy "Server" interfaces which do not guarantee backward compatibility`)
+	requireUnimplemented = flags.Bool("require_unimplemented_servers", true, "set to false to match legacy behavior")
 
 	protogen.Options{
 		ParamFunc: flags.Set,
 	}.Run(func(gen *protogen.Plugin) error {
-		gen.SupportedFeatures = uint64(pluginpb.CodeGeneratorResponse_FEATURE_PROTO3_OPTIONAL)
```

Comment:
This should be kept as this means that the tool supports optional parameters

Reply:
Ack, thanks for catching that.  I'll re-add shortly.